msdsn/conn_str.go

```diff
@@ -651,16 +659,6 @@ func normalizeOdbcKey(s string) string {
 	return strings.ToLower(strings.TrimRightFunc(s, unicode.IsSpace))
 }
 
-const defaultServerPort = 1433
-
-func resolveServerPort(port uint64) uint64 {
```

Comment:
my IDE flagged it as "unused", so I applied "The Boy Scout Rule"

Reply:
yeah it moved to protocol.go thx